languages/csharp/exercises/concept/overflow/Overflow.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+public static class CentralBank
+{
+    public static string DisplayDenomination(long @base, long multiplier)
```

Comment:
Note: we have to explain the `@` prefix for identifiers somewhere in a prerequisite exercise. Might be something to add to the `after.md` of the `basics` exercise.

Reply:
Issue #1979 addresses this.